pydantic_duality/__init__.py

```diff
@@ -58,11 +74,11 @@ def _alter_attrs(attrs: dict[str, object], name: str, attr: str):
             if attr == PATCH_REQUEST_ATTR:
                 if get_origin(annotations[key]) is Annotated:
                     args = get_args(annotations[key])
-                    annotations[key] = Annotated.__class_getitem__(tuple([args[0] | None, *args[1:]]))
+                    annotations[key] = Annotated.__class_getitem__(tuple([Optional[args[0]], *args[1:]]))
                 elif isinstance(annotations[key], str):
-                    annotations[key] += " | None"
+                    annotations[key] = f"Optional[{annotations[key]}]"
```

Comment:
I'm not sure exactly about this line and what if Optional is not defined, but seems like it should work.